lib/evil_seed/record_dumper.rb

```diff
@@ -50,14 +50,21 @@ def transform_and_anonymize(attributes)
       end
     end
 
+    def insertable_column_names
+      model_class.columns_hash.reject { |k,v| v.virtual? }.keys
+    end
+
     def insert_statement
       connection = model_class.connection
       table_name = connection.quote_table_name(model_class.table_name)
-      columns    = model_class.column_names.map { |c| connection.quote_column_name(c) }.join(', ')
+      columns    = insertable_column_names.join(', ')
```

Comment:
I believe it is better to keep quoting or it may fail on columns named the same as reserved SQL keywords.